ansible/roles/builds/templates/github_ssh_config.j2

```diff
@@ -1,4 +1,4 @@
-host github.com
- HostName github.com
+host github
+ HostName params.GITHUB_REPOS_BASE_URL
```

Comment:
This doesn't look like it would work, am I missing something? `params` is a Jenkins variable isn't it?